content/en/docs/configuration/acme/dns01/route53.md

```diff
@@ -225,6 +225,33 @@ metadata:
     eks.amazonaws.com/role-arn: arn:aws:iam::XXXXXXXXXXX:role/cert-manager
 ```
 
+Make sure there is no `role: arn...` field on the issuer definition, otherwise you might end up with `sts:assumerole` permission error. The issuer or clusterissuer resource definition in case of EKS OIDC should look a like this:
```

Comment:
```suggestion
Make sure there is no `eks.amazonaws.com/role-arn` annotation on the issuer definition, otherwise you might end up with `sts:assumerole` permission error. The issuer or clusterissuer resource definition in case of EKS OIDC should look a like this:
```

Reply:
Thank you for the suggestion @meyskens !

I modified wording and fixed a few more typos. Hopefully looks better now, ty!